config/freeradius3/mods-perl-privacyidea

```diff
@@ -1,3 +0,0 @@
-perl {
-    filename = /usr/share/privacyidea/freeradius/privacyidea_radius.pm
-}
```

Comment:
I would like to keep the file name indicating that this is a module.
Please rename the file to contain mods or modules.

Reply:
fixed in 82d037a